File: Data Structures & Algorithms/is-anagram/submission-6.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        if(s.Length!=t.Length)
        {
            return false;
        }
        int[] freqs=new int[26];
        int[] freqt=new int[26];
        for(int i=0;i<s.Length;i++)
        {
            freqs[s[i]-'a']++;
            freqt[t[i]-'a']++;
        }
        for(int i=0;i<26;i++)
        {
            if(freqs[i]!=freqt[i])
            {
                return false;
            }
        }
        
        return true;
    }
}

File: Data Structures & Algorithms/is-subsequence/submission-6.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        i=0
        j=0
        if len(s)>len(t):
            return False
        while i<len(s) and j<len(t):
            if s[i]==t[j]:
                i+=1
            j+=1
        return i==len(s)
        